contracts/LifMarketMaker.sol

```diff
@@ -63,21 +68,25 @@ contract LifMarketMaker is Ownable {
     totalPeriods = _totalPeriods;
     foundationAddr = _foundationAddr;
     initialWei = msg.value;
-    initialBuyPrice = initialWei.div(lifToken.totalSupply());
+    initialBuyPrice = initialWei
+      .div(lifToken.totalSupply().div(PRICE_FACTOR))
```

Comment:
we should first `mul(PRICE_FACTOR)` and then `div(lifToken.totalSupply....)` to keep more precision